app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -0,0 +1,61 @@
+module PublishingApi
+  class MinistersIndexPresenter
+    attr_accessor :item
+    attr_accessor :update_type
+
+    def initialize(item: nil, update_type: nil)
```

Comment:
I don't think we even need to accept an `item` parameter because it will never be set to anything meaningful. Then we can remove the `item` `attr_accessor`.